build/_k-standard-goals.shade

```diff
@@ -65,7 +65,19 @@ default Configuration='${E("Configuration")}'
   k-restore
 
 #target-dir-clean target='clean'
-  rimraf rimrafDir='${TARGET_DIR}' if='Directory.Exists(TARGET_DIR)'
+  @{
+      if(Directory.Exists(TARGET_DIR))
+      {
+          var directory = new DirectoryInfo(TARGET_DIR) { Attributes = FileAttributes.Normal };
+
+          foreach (var info in directory.GetFileSystemInfos("*", SearchOption.AllDirectories))
+          {
+              info.Attributes = FileAttributes.Normal;
```

Comment:
The system won't let you change some files to `Normal` but would let you remove them. So my `|=` suggestion should have been `&= ~FileAttributes.ReadOnly`. `ReadOnly` is the only bit worth twiddling here.